.circleci/config.yml

```diff
@@ -5,7 +5,20 @@
         image: ubuntu-1604:201903-01
       steps:
         - checkout 
-        - run: 'make'
+        - run:
+            name: 'requirements'
+            command: |
+              sudo apt-get -y -q update
+              sudo apt-get install graphviz
+              sudo apt-get install doxygen
+              make
+        - run:
+            name: 'Doxygen'
+            command: |
+              doxygen -g Doxyfile;
+              doxygen Doxyfile;
+        - store_artifacts:
+            path: ./html
```

Comment:
Ha nem deployoljatok valahova, akkor a latex-es pdf-et is artifactoljatok, vagy csak azt.